frontend/pages/index.tsx

```diff
@@ -1,16 +1,228 @@
-import { NextPage } from "next";
-import LandingPageLayout from "../components/Layouts/LandingPageLayout";
+import { NextPage } from 'next'
```

Comment:
It seems like we're thinking the same thing :) I'll update the code to slim down `pages/index.js`. 

The only thing is that I had to not use the `LandingPageLayout` because it essentially didn't do anything. The components need to take up 100% width, but internal to the component, have a max-width of 1200px. This is what the original pages had, and there isn't really a way of doing it other than that (unless there's something obvious I'm missing)